db/dbHelpers.js

```diff
@@ -41,6 +41,14 @@ const addAppointment = (data, callback) => {
   });
 };
 
+const removeAppointment = (data, callback) => {
+  const queryString = `DELETE FROM appointments_log WHERE id=${data.appointment_id} AND business_id=${data.business_id}`;
```

Comment:
I don't know if this is necessary but there might be a constraint you can add here that assures that the record is only deleted if it exists (not sure how error handling works in SQL)